go.mod

```diff
@@ -8,17 +8,17 @@ require (
 	github.com/armon/go-metrics v0.4.1
 	github.com/cosmos/cosmos-sdk v0.46.13-rc.0
 	github.com/cosmos/ibc-go/v5 v5.2.1
-	github.com/crypto-org-chain/cronos/memiavl v0.0.1
-	github.com/crypto-org-chain/cronos/store v0.0.0-20230425151210-b262fa8bdacb
-	github.com/crypto-org-chain/cronos/versiondb v0.0.0
+	github.com/crypto-org-chain/cronos/memiavl v0.0.2
+	github.com/crypto-org-chain/cronos/store v0.0.2
+	github.com/crypto-org-chain/cronos/versiondb v0.0.0-00010101000000-000000000000
 	github.com/ethereum/go-ethereum v1.10.26
-	github.com/evmos/ethermint v0.6.1-0.20221101220534-a8ea4eceb6d9
+	github.com/evmos/ethermint v0.0.0-00010101000000-000000000000
 	github.com/gogo/protobuf v1.3.3
 	github.com/golang/protobuf v1.5.3
 	github.com/gorilla/mux v1.8.0
 	github.com/grpc-ecosystem/grpc-gateway v1.16.0
 	github.com/linxGnu/grocksdb v1.7.16
-	github.com/peggyjv/gravity-bridge/module/v2 v2.0.0-20220420162017-838c0d25e974
```

Comment:
not sure if this bridge is need?

Reply:
not needed anymore since the testnet is taken down, we can cleanup the code in another PR, or just leave it there.